pkg/cluster/internal/create/actions/kubeadminit/init.go

```diff
@@ -143,6 +141,33 @@ func (a *action) Execute(ctx *actions.ActionContext) error {
 	return nil
 }
 
+// getHostPort retrives the random host where the API server/the control plane is exposed
+func getHostPort(allNodes []nodes.Node) (int, error) {
```

Comment:
:facepalm: I knew this problem was familiar in #418 ...

Thanks for this. Sorry it didn't get in sooner.